src/v/cluster/archival/tests/BUILD

```diff
@@ -89,3 +118,203 @@ redpanda_cc_gtest(
+        "//src/v/cloud_storage_clients",
+        "//src/v/cluster",
+        "//src/v/config",
+        "//src/v/model",
+        "//src/v/net",
+        "//src/v/ssx:sformat",
+        "//src/v/storage",
+        "//src/v/storage:resources",
+        "//src/v/storage/tests:disk_log_builder",
+        "//src/v/test_utils:archival",
+        "//src/v/test_utils:fixture",
+        "//src/v/test_utils:scoped_config",
+        "//src/v/utils:retry_chain_node",
+        "//src/v/utils:unresolved_address",
+        "@boost//:test",
+        "@seastar",
+        "@seastar//:testing",
+    ],
+)
+
+redpanda_cc_btest(
+    name = "ntp_archiver_reupload_test",
+    timeout = "short",
+    srcs = [
+        "ntp_archiver_reupload_test.cc",
+    ],
+    cpu = 1,
+    deps = [
+        ":service_fixture",
+        "//src/v/cloud_storage",
+        "//src/v/cloud_storage_clients",
+        "//src/v/cluster",
+        "//src/v/config",
+        "//src/v/storage",
+        "//src/v/test_utils:fixture",
+        "//src/v/test_utils:scoped_config",
+        "@seastar",
+    ],
+)
+
+redpanda_cc_btest(
+    name = "retention_strategy_test",
+    timeout = "short",
+    srcs = [
+        "retention_strategy_test.cc",
+    ],
+    cpu = 1,
+    deps = [
+        ":service_fixture",
+        "//src/v/cluster",
+        "//src/v/storage/tests:disk_log_builder",
+        "//src/v/test_utils:archival",
+        "//src/v/test_utils:seastar_boost",
+        "@seastar",
+        "@seastar//:testing",
+    ],
+)
+
+redpanda_cc_btest(
+    name = "async_data_uploader_test",
+    timeout = "short",
+    srcs = [
+        "async_data_uploader_test.cc",
+    ],
+    cpu = 1,
+    deps = [
+        "//src/v/bytes:iostream",
+        "//src/v/cluster",
+        "//src/v/config",
+        "//src/v/kafka/client",
+        "//src/v/kafka/server/tests:kafka_test_utils",
+        "//src/v/model",
+        "//src/v/random:generators",
+        "//src/v/redpanda/tests:fixture",
+        "//src/v/storage",
+        "//src/v/test_utils:fixture",
+        "//src/v/test_utils:seastar_boost",
+        "@boost//:test",
+        "@seastar",
+        "@seastar//:testing",
+    ],
+)
+
+redpanda_test_cc_library(
+    name = "archival_service_fixture",
+    hdrs = ["archival_service_fixture.h"],
+    include_prefix = "cluster/archival/tests",
+    deps = [
+        ":service_fixture",
+        "//src/v/cloud_storage_clients",
+        "//src/v/cluster",
+        "//src/v/cluster/tests:cluster_test_fixture",
+        "//src/v/config",
+        "//src/v/container:fragmented_vector",
+        "//src/v/http/tests:utils",
+        "//src/v/model",
+        "//src/v/random:generators",
+        "//src/v/test_utils:fixture",
+        "//src/v/test_utils:seastar_boost",
+        "@boost//:test",
+        "@seastar",
+    ],
+)
+
+redpanda_cc_btest(
+    name = "archiver_manager_test",
+    timeout = "long",
+    srcs = [
+        "archiver_manager_test.cc",
+    ],
+    memory = "1028MiB",
```

Comment:
The default is 1G so should this be 2G?

Reply:
ah, thought the default was lower. perhaps doesn't need adjusting at all then...

Reply:
eh, tuned to 2G and it seems to work. I'm leavin it